temporalcli/commands.gen.go

```diff
@@ -1556,6 +1557,7 @@ func NewTemporalTaskQueueDescribeCommand(cctx *CommandContext, parent *TemporalT
 	s.TaskQueueTypeLegacy = NewStringEnum([]string{"workflow", "activity"}, "workflow")
 	s.Command.Flags().Var(&s.TaskQueueTypeLegacy, "task-queue-type-legacy", "Task Queue type (legacy mode only). Accepted values: workflow, activity.")
 	s.Command.Flags().IntVar(&s.PartitionsLegacy, "partitions-legacy", 1, "Query for all partitions up to this number (experimental+temporary feature) (legacy mode only).")
+	s.Command.Flags().BoolVar(&s.DisableStats, "disable-stats", false, "Disable task queue statistics.")
```

Comment:
@ShahabT  - Since this has now been changed to `disable-stats` with a default value of false, it brings us back to the bug we found in matching where a versioned queue may not be loaded leading to null dereferences. Again, not a big deal as you said since versioning happens to be pre-release but just wanted to make it a note here.

Reply:
Outdated comment since a patch was merged to main last week - [PR](https://github.com/temporalio/temporal/pull/6437)